### DISCUSSION TITLE
Hide projects in RBAC

### DISCUSSION THREAD

**Discussion by Shubham1406220** [argoproj/argo-cd#13152] [General]
By default argocd gives read-only role. Is there a way we can hide projects from UI. If there is already a link for the same that would also help

**Comment by jannfis** [comment]
By default, Argo CD gives an authenticated user whatever is specified as the `policy.default` field of the `argocd-rbac-cm` ConfigMap.

You can either remove that field or set it to the empty string, or specify a custom role to be applied by default.

**Reply by Shubham1406220** [reply]
Thanks for the help, we have added a default read-only policy after removing that and giving the permission explicitly we could hide the non-required projects.
